src/main/java/org/bytecodeparser/instruction/InstructionArguments.java

```diff
@@ -0,0 +1,22 @@
+package org.bytecodeparser.instruction;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class InstructionArguments {
+
+    private InstructionTypes type;
+
+    private byte indexByte;
+    private short indexShort;
+    private byte count;
+    private byte zero;
```

Comment:
there can be multiple zeros

Reply:
I haven't seen any examples of it. 
My proposition: when someone will be add this case to InstructionTypes, he should resolve "multiple zeros" problem in InstructionArguments.

Reply:
Do you understand purpose of zero operand at all? Could you explain it?

Reply:
it is used by InvokeDynamic - in runtime this instruction is substituted with another one, maybe tha's why there are extra bytes